tests_integration/nativeapp/test_version.py

```diff
@@ -355,15 +363,21 @@ def test_nativeapp_version_create_package_no_magic_comment(
 
 # Tests a simple flow of an existing project, executing snow app version create, drop and teardown, all with distribution=internal
 @pytest.mark.integration
+@pytest.mark.parametrize(
+    "create_command,list_command,drop_command,test_project",
+    [["app version create", "app version list", "app version drop", "napp_init_v2"]],
+)
```

Comment:
I assume this is for consistency, since there doesn't seem to be a need to use a parametrized test here?

Reply:
yeah, I wanted to keep the parametrization for two reasons, 1. it makes this PR's diff smaller, and 2. it puts the relevant info in the test signature rather than in the body, keeping the test IDs consistent for stuff like snapshot names, debugging, etc